python/pyspark/ml/param/__init__.py

```diff
@@ -170,6 +170,15 @@ def toVector(value):
         raise TypeError("Could not convert %s to vector" % value)
 
     @staticmethod
+    def toMatrix(value):
+        """
+        Convert a value to ML Matrix, if possible
+        """
+        if isinstance(value, Matrix):
+            return value
```

Comment:
Is this method really necessary?  It's not actually converting anything, just checking the type.  If this wasn't here and the user put something other than a Matrix, what error would be raised?